### DISCUSSION TITLE
What about rules severity ?

### DISCUSSION THREAD

**Discussion by augustinde** [green-code-initiative/ecoCode#275] [General]
Hi, all the rules are classified as "minor", I wanted to know if this classification would be subject to change? Aren't some rules more impactful than others?

**Comment by dedece35** [comment]
Hi,

@glalloue, @olegoaer ? any point of view ? 😜

**Comment by olegoaer** [comment]
@augustinde If you look at the Android plugin, all not rules are equals. So the general answer is: no, each rule has a specific severity, but in the absence of a precise evaluation, it is left at "minor" by default.

**Reply by augustinde** [reply]
Thanks for your answer !
